Content.Shared/Clothing/EntitySystems/ToggleableClothingSystem.cs

```diff
@@ -110,10 +113,15 @@ private void StartDoAfter(EntityUid user, EntityUid item, EntityUid wearer, Togg
         if (!_doAfter.TryStartDoAfter(args))
             return;
 
-        if (!stealth)
+        if (!hidden)
         {
-            var popup = Loc.GetString("strippable-component-alert-owner-interact", ("user", Identity.Entity(user, EntityManager)), ("item", item));
-            _popupSystem.PopupEntity(popup, wearer, wearer, PopupType.Large);
+            PopupType? popupSize = _thieving.GetPopupTypeFromStealth(stealth);
+
+            if (popupSize.HasValue) // we should always have a value if we're not hidden
+                _popupSystem.PopupEntity(Loc.GetString("strippable-component-alert-owner-interact",
+                ("user", (stealth == ThievingStealth.Subtle) ? "Someone" : Identity.Entity(user, EntityManager)),
+                ("item", item)),
+                wearer, wearer, popupSize.Value);
```

Comment:
A fourth in another file?

Reply:
Looking back I see how these are slightly different, whatever, just do the original requested changes.